pkg/utils/errors.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"fmt"
+)
+
+type Errors struct {
+	errors []error
+}
+
+func (e *Errors) Errors() string {
+	return "TEST"
+}
+
+func New(text string) *Errors {
+	e := &Errors{}
+	e.errors = make([]error, 10)
+	err := fmt.Errorf("Error chain %s", text)
+	e.errors = append(e.errors, err)
+	return &Errors{errors: e.errors}
+}
+
+func (e *Errors) Add(err error) {
+	if e.errors == nil {
+		e.errors = make([]error, 10)
+	}
+	e.errors = append(e.errors, err)
+	//	return e
```

Comment:
Remove comment ? we won't need to think of returning. Is a little confusing.

Reply:
resolved